cli/internal/run/run.go

```diff
@@ -427,9 +436,10 @@ but don't actually run them. Passing --dry=json or
 )
 
 func addRunOpts(opts *runOpts, flags *pflag.FlagSet, aliases map[string]string) {
-	flags.StringVar(&opts.dotGraph, "graph", "", "Generate a Dot graph of the task execution.")
+	flags.StringVar(&opts.dotGraph, "graph", "", "Generate a graph of the task execution.")
 	flags.IntVar(&opts.concurrency, "concurrency", 10, "Limit the concurrency of task execution. Use 1 for serial (i.e. one-at-a-time) execution.")
 	flags.BoolVar(&opts.parallel, "parallel", false, "Execute all tasks in parallel.")
+	flags.StringVar(&opts.dryRun, "dry-run", "", _dryRunHelp)
```

Comment:
Unfortunately, this changes the usage text for this flag from:

`--dry-run / dry [=text|json]` to `--dry-run string[="text"]`

We're losing the alias and the set of options. Similarly for the new options you've added, the usage text is `--graph string[="dotGraph.internal"]` which centers the one value we don't need them to use. 

There are a couple paths forward here. 
 * You can implement a custom `pflag.Value` for `--graph`.
 * You can try implementing a generic "boolean or optional values that maps to an enum" pflag.Value. I don't know if this is possible with Go, but maybe!
 * You can customize via `SetUsageTemplate` or `SetHelpTemplate` or both or even the associated `SetHelpFunc` and `SetUsageFunc` for `run`. Even if we don't do this now, I don't think this path should be dismissed, since it's the UX for `turbo` and doing custom UX work to make `turbo` easier and friendlier to use seems worth it to me. We definitely should not feel beholden to `pflag`'s defaults.

Reply:
Ah good catch on the help text. I missed that in my testing. 

I can definitely go back to defining the custom args like were being done for both of these - this just looked like a much simpler method to achieving the same result. But now I see why you did it in the first place 😄 

I'll take a look at the custom help text options as well when revisiting this.